spec.html

```diff
@@ -36755,6 +36755,17 @@ <h1>Structured Data</h1>
   <emu-clause id="sec-arraybuffer-objects">
     <h1>ArrayBuffer Objects</h1>
 
+    <emu-clause id="sec-arraybuffer-notation">
+      <h1>Notation</h1>
+      <p>The descriptions below in this section, <emu-xref href="#sec-atomics-object"></emu-xref>, and <emu-xref href="#sec-memory-model"></emu-xref> use the read-modify-write modification function internal data structure.</p>
+      <p>A <em>read-modify-write modification function</em> is a mathematical function that is notationally represented as an abstract closure that takes two Lists of bytes as arguments returns a List of bytes. These abstract closures satisfy all of the following properties:</p>
+      <ul>
+        <li>They do not access any language or specification values except their parameters, captured values, and the [[LittleEndian]] field of the surrounding agent's Agent Record.</li>
```

Comment:
They don't ever refer to `[[LittleEndian]]`, that I can see.

Reply:
Oops, will remove.